packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +194,80 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents IContentResponse parameters.
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared */
+      threadId: string;
```

Comment:
Can thread IDs really be any string or is there a format to them? Is "Shanmathi" a valid thread ID?

Reply:
No, threadIds are GUIDs that are returned from Teams side.

Reply:
Same comment as for `author`, then. Thanks!

Reply:
Added validation and updated documentation.